packages/cli/src/commands/buildHandler.js

```diff
@@ -39,6 +40,8 @@ export const handler = async ({
   const useFragments = rwjsConfig.graphql?.fragments
   const useTrustedDocuments = rwjsConfig.graphql?.trustedDocuments
 
+  const require = createRequire(path.join(getPaths().base, 'node_modules'))
```

Comment:
I'll probably refactor this into a helper once I have a better idea of where it's used